gemp-lotr/gemp-lotr-cards/src/main/resources/cards/unofficial/pc/setV02/setv02-Gandalf-playtest.hjson

```diff
@@ -168,15 +196,27 @@
 		}
 		*/
 		effects: {
-			type: event
+			type: activated
```

Comment:
This should be left as "type: event", as "activated" is an ability on a card in play that starts with "Maneuver: " or some other phase.